File: fibonacciSeries/fibonacciSeries.py
num = int(input("Enter the Nth number upto which series you want to see"))
a=0
b=1
print(a," , ",b," , ",end="")
for i in range(2,num+1):
    nextTerm = a+b
    a=b
    b=nextTerm
    if i==num:
        print(nextTerm)
    else:
        print(nextTerm,end=" , ")

File: fibonacciSeries/fibonacciSeries.java
public class fibonacciSeries{
    public static void main(String[] args) {
        int num = 7;
        int a=0,b=1;
        System.out.print(a+" , "+b+" , ");
        int nextTerm;
        for(int i=2;i<=num;i++){
            nextTerm=a+b;
            a=b;
            b=nextTerm;
            if(i==num){
                System.out.print(nextTerm);
            }
            else{
                System.out.print(nextTerm+" , ");
            }
        }
    }
}